### DISCUSSION TITLE
Aesthetic

### DISCUSSION THREAD

**Discussion by erlend-sh** [muni-town/weird#21] [Ideas]
Weird pages will have many distinct styles and they’ll be as customizable as any web page. The Weird *platform service* however needs a unique aesthetic.

The retro look is appealing to me:

![image](https://github.com/commune-os/weird/assets/583842/9c3c52b3-4efd-4947-be50-b4f12260f665)

![image](https://github.com/commune-os/weird/assets/583842/c315791f-bd33-4791-8a78-8f4d647457ef)

![](https://images.vexels.com/media/users/3/174808/raw/b4239d6627be0d570533bb202e5c4f49-90s-internet-aesthetic-poster-template.jpg)

![](https://thumbs.dreamstime.com/z/world-wide-web-illustration-7992757.jpg)

https://www.freepik.com/vectors/retro-ui

Not that the UI would be set up like a virtual desktop OS though.

**Comment by zicklag** [comment]
That's kind of fun. I think I can do styling pretty well when I have something to copy for inspiration. :grin:

**Comment by erlend-sh** [comment]
As for people's own websites, we will of course allow just about any kind of style, but I would love to provide some good defaults (and sort of set a convention) with fully painted backgrounds:

https://www.magicpattern.design/tools/css-backgrounds
https://css-pattern.com/
https://github.com/bansal/pattern.css

Weird urges people to *take up some space* on the web. Filled backgrounds give websites that *chunky* feel, like they have some weight, and they're not just floating around in white/black space.

**Comment by zicklag** [comment]
Brainstorming aesthetic, my bro mentioned that the retro screenshot you shared reminds him of Pop!_OS's aesthetic a little bit: https://pop.system76.com/

I'm also thinking from a juicing standpoint it'd be great to have fun animations and stuff wherever appropriate, and I've seen sites that use this to to really increase the feel of their page.

Some references:

- https://www.drone.io/
- https://12wave.com/
- https://www.duolingo.com/

As for how to render those animations, we should look into Lottie: https://lottiefiles.com/.

Lottie doesn't necessarily have a lot of Open Source creators, though there are some with free plans/tirals or something. This is an Open Source creator we could try: https://glaxnimate.mattbas.org/.

In addition to Lottie, we should check out https://gsap.com/, which is less for icons/dioramas and more for site interactivity, so we might combine it with with Lottie/SVG animations.

**Reply by erlend-sh** [reply]
Yeah, all of that! We just gotta be very incremental about adding this stuff.

https://garden.bradwoods.io/notes/design/juice

**Reply by zicklag** [reply]
Just noticed that GSAP requires a separate commercial license, so might not want to use that yet.

Lennis could also be useful: https://lenis.darkroom.engineering/

**Comment by zicklag** [comment]
Another reference off the [Lottie](https://lottiefiles.com) website for a possibly good retro aesthetic.

![camera](https://github.com/commune-os/weird/assets/25393315/504c3f24-c094-4005-82f1-71b213606005)

**Reply by erlend-sh** [reply]
Yeah I love that! Any idea what terms to search with for more of that style?

I also added some new ref pics up top 👆

**Reply by zicklag** [reply]
> Any idea what terms to search with for more of that style?

Completely clueless, :upside_down_face: Got lucky with that one, happened to be right on the frontpage for the Lottie website. I've noticed it's hard to search for.

**Comment by erlend-sh** [comment]
You know what’d be properly *weird*? If the website had two styles. They don’t have to be super different, nothing crazy. But two distinctly different styles nonetheless. And as you’re browsing weird.one/* you will occasionally see intentional *glitches* that reveal the other styling.

And upon every new visit, one of the two themes are gonna be picked for you at random. It’s an aesthetic gimmick in line with our application’s nature as a multi-site/verse.

v2 / v3 kind of stuff though.

**Reply by zicklag** [reply]
Interesting thought. I'm all for it as long as it doesn't feel annoying in practice, to the users. But if it was subtle enough, it might not.

Not to get too in line with the dumb "make it new" things that games like Fortnight do, but it'd be fun to do maybe seasonal or rotating styles, too.

**Comment by erlend-sh** [comment]
For a darker theme, the retro-neon side of vaporwave greatly appeals to me:

![vapor1](https://github.com/commune-os/weird/assets/583842/e9d38c0b-92a7-47b4-bdac-aa809cf620ae)
![vapor2](https://github.com/commune-os/weird/assets/583842/75b354b4-c6ae-48cd-97dd-eac3c1227d45)
![vapor4](https://github.com/commune-os/weird/assets/583842/8f03f6ed-227e-4983-a9f3-4d7f3ab4ff4c)

https://webflow.com/made-in-webflow/website/Lars-Olson-Designs-Stuff

![IMG_2367](https://github.com/commune-os/weird/assets/583842/05dd1439-22aa-4689-925e-7c8ffc8c3f0c)
![IMG_2368](https://github.com/commune-os/weird/assets/583842/630e3968-2b05-4bf2-9462-70fdd3340302)


https://webflow.com/blog/retrofuturism
